frontend/src/lib/components/icons.tsx

```diff
@@ -30,6 +30,72 @@ export function IconJavascript({ style }: { style?: CSSProperties }): JSX.Elemen
     )
 }
 
+export function IconNodeJS({ style }: { style?: CSSProperties }): JSX.Element {
+    return (
+        <svg width="1em" height="1em" viewBox="0 0 256 282" xmlns="http://www.w3.org/2000/svg" style={style}>
```

Comment:
As mentioned in the PR desc, I used arbitrary svg icons, please let me know if you want to suggest your own svgs